andela/ah-shakas
# #160577476 Users Verification after signup

### What does this PR do?

-  Verify user after signup
### Description of Task to be completed?
- The users can verify their account after signup.
- A link is sent to the email. After the user clicks the link they will be able to activate their account.
 `Note: Before clicking the link, the account is inactive`
### How should this be manually tested?
- Clone [ah-shakas](https://github.com/andela/ah-shakas) repo `$ git clone https://github.com/andela/ah-shakas.git`
- export db credentials. please note that there is no spaces before and after `=`

```
      export DB_NAME="name of your database"
      export DB_PASSWORD="passord to your database user"
      export DB_HOST="your local server eg (127.0.0.1)"
      export DB_USER="your database username eg (postgres)"
```
_Note- if you have no password set for your database user leave the password field blank i.e `export DB_PASSWORD=""`_
- On the project's root directory run `python manage.py runserver`
- send a `POST` request to the server http://127.0.0.1:8000/users/ whith the following in the body.
```
{
  "user":{
    "email":"email@domain.com",
    "username":"simple username",
    "password":"AlphaNumericPassword"
  }
}
```
- Done! 🙂
### What are the relevant pivotal tracker stories?
- https://www.pivotaltracker.com/story/show/160577476
